aptos-move/aptos-vm-types/src/tests/test_change_set.rs

```diff
@@ -57,7 +58,22 @@ use std::collections::HashMap;
 /// *--------------*----------------*----------------*-----------------*
 /// ```
 
-macro_rules! write_set_1 {
+macro_rules! resource_write_set_1 {
+    ($d:ident) => {
+        vec![
+            mock_create_with_layout(format!("0{}", $d), 0, None),
```

Comment:
Not relevant, but I ve been also modifying these things and am very tempted to make vmchangeset generic? This can make it very easy to fuzz/test. cc @gelash @igor-aptos @vusirikala